css/styles.css

```diff
@@ -1,5 +1,16 @@
 /* basic styles */
 
+* {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  border: 0 none;
+  list-style: none outside none;
+  margin: 0;
+  outline: 0 none;
+  padding: 0;
+}
```

Comment:
- [ ] Bootstrap already "normalize" the browser default stylesheet this is not necesary. Please Remove.